### ISSUE TITLE
Test scaffolding command or A/A test generation

### ISSUE THREAD

**Issue by kingo55** [mint-metrics/mojito-js-delivery#8]
To make it easy for new users to get started, we could have a test scaffolding command to create a `live` A/A test in the waves directory:

```yml
id: aa1
name: AA test 1
sampleRate: 1
state: live
trigger: trigger.js
recipes:
  0:
    name: A1
  1:
    name: A2
```

```js
function trigger(test)
{
  test.activate();
}
```

A simple gulp command would generate the files needed:

```
gulp scaffold
```

Not attached to this particular naming - might be a better term we could use here.

**Comment by allmywant** [comment]
@kingo55 Good idea!

**Comment by dapperdrop** [comment]
I like this. We could also have multiple scaffold templates setup to showcase some simple experiments / triggers. I.e. prefill some treatment JS files with innocuous code, like printing something to the console. Users can then use them as a launch pad to their own experiments.

E.g.
- scaffold demo "A/B console". Control: does nothing, Treatment: outputs something in the console
- scaffold demo "A/B css". Control: does nothing, Treatment: changes `<p> letter-spacing` by a small amount
- etc

**Comment by kingo55** [comment]
Feels like we also need a scaffold for new tests we're setting up too. It'll save the 15 seconds creating the test YAML and include files:

```
$ gulp scaffold w212
```

Then it creates all the necessary files:

 - `w212/`
    - `config.yml`
    - `trigger.js`
    - `1.js`
    - `1.css`
    - `shared.js`
    - `shared.css`

And for scaffold demo/AA tests we could try something like this:

```
$ gulp scaffold-demo
$ gulp scaffold-aa
```

**Comment by dapperdrop** [comment]
That would be handy! Not sure if we should scaffold shared files as it's not a common pattern and would more than likely result in users deleting them in the setup process. 

I reckon MVP would be:
```
config.yml
trigger.js
```
And probably `1.js`

**Comment by kingo55** [comment]
Closed via #19
